src/app/sections/ProbeDetails.svelte

```diff
@@ -20,6 +21,10 @@ function probeIsSelected(probe) {
   return probe.name !== null && probe.name !== 'null';
 }
 
+function getProbeDictURL(apiName) {
```

Comment:
this is totally fine, but this can also be down below with 

```<a href="https://probes.telemetry.mozilla.org/?view=detail&probeId={$store.probe.apiName}">blahblahblah</a>```

I tend to do this since it removes a layer of indirection

Reply:
The link logic had some branching before robhudson gave us the key in `apiName` so yes this is an old fragment.